README.md

```diff
@@ -18,9 +18,14 @@ A source connector pulls data from an external resource and pushes it to downstr
 
 ### Configuration
 
-| name                  | description                           | required | default value |
-|-----------------------|---------------------------------------|----------|---------------|
-| `source_config_param` | Description of `source_config_param`. | true     | 1000          |
+| name       | description                                            | required | default value |
+| ---------- | ------------------------------------------------------ | -------- | ------------- |
+| `host`     | The hostname or IP address of the MySQL server         | true     |               |
+| `port`     | The port number on which the MySQL server is listening | false    | 3306          |
+| `user`     | The MySQL user account name for authentication         | true     |               |
+| `password` | The password for the MySQL user account                | true     |               |
+| `database` | The name of the specific MySQL database to connect to  | true     |               |
+| `tables`   | The list of tables to pull data from                   | true     |               |
```

Comment:
A question out loud. Is adding a wildcard such as `*` (being that one as default value and not required) something doable? If it adds too much scope, we could start by creating an issue to make sure we keep track of that.

Reply:
Sorry I did not understand you @raulb. What do you mean with the wildcard usage?

Reply:
@alarbada I mean that if you provide `*` on the tables configuration fields, you would pull data from all tables for example.

Reply:
As discussed during today's sync. Let's create an issue for this so we can track separately! (thank you @alarbada 🙏 )

Reply:
Just created it @raulb, it's this one #5 

As this connector might be the most complicated one I've made in a while, I first want to figure out the MVP version of the connector so that I get an idea on how it works e2e. I'd do this feature after I get the acceptance tests working.

I'm sure it is a very useful feature nevertheless.